src/pages/Profile/Profile.css

```diff
@@ -13,6 +13,60 @@
          This will help you when you get started.
     */
 
+.text {
+    /* flex-direction: column; */
+    margin: auto;
+    width: 800px;
+}
+
+#circle img {
```

Comment:
generally it's not very good practice to directly select tags either, if we need to style images we should apply a unique `className` to them based on how far we want to generalize the styles